en/user_guide/workflow.rst

```diff
@@ -5,21 +5,23 @@ In Open Orchestra, key contributions such as nodes and contents are subject to a
 It allows several levels of validation by different people from creation to publication. Workflow
 are fully customizable.
 
-A workflow is composed of steps, each corresponding to a status. Each status is linked to one (or
-several) other status by a transition. To change a content status or a node status, a contributor
-must have a right mapping the corresponding transition.
+A workflow is composed of steps, each corresponding to a status.
+Each status could be linked to an other by a transition named 'Role'.
+These roles are grouped in collection named 'Workflow Function'.
+Finally, in user's administration, it is possible to add a workflow function to a user for a given content type or for nodes.
 
-For instance, the article content type could have a workflow represented by the following status:
+For example :
+    - three statuses : draft, pending and published
+    - two roles : DRAFT_TO_PUBLISHED (transition draft to published), PENDING_TO_PUBLISHED (transition pending to published)
+    - one workflow function: Validator (containing DRAFT_TO_PUBLISHED and PENDING_TO_PUBLISHED)
+    - a user with Validator function on News (content type)
```

Comment:
You should use `` in this paragraph to distinguish the names of your example: ``draft``, ``DRAFT_TO_PUBLISHED``, ``validator`` and so on